README.md

```diff
@@ -88,3 +88,32 @@ Get a resource from Neo4j.
 
 ## Code Documentation
 To document new files, please check [here](https://github.com/release-engineering/estuary-api/tree/master/docs).
+
+
+## Authorization
+
+If authentication is enabled, Estuary can authorize users based on their employee type and a user
+whitelist configured through the membership of an LDAP group.
+
+### Employee Type
+
+You may set the list of valid employee types with the configuration item `EMPLOYEE_TYPES`. These
+employee types map to the `employeeType` LDAP attribute of the user that is added to the OpenID
+Connect token received by Estuary.
+
+### Configuring the Whitelist
+
+To configure a whitelist of users, they must be part of an LDAP group configured with Estuary. The
+following configuration items are required:
+
+* `LDAP_URI` - the URI to the LDAP server to connect to in the format of
+    `ldaps://server.domain.local`.
+* `LDAP_EXCEPTIONS_GROUP_DN` - the distinguished name to the LDAP group acting as the whitelist.
```

Comment:
What do you mean by 'distinguished name'? Is this something users will generally understand what you are referring to, since it could just be me not knowing.

Reply:
It's a string that uniquely identifies an entry in the LDAP directory. It's an implementation detail that the user won't be aware of.

Reply:
Oops! By users I mean like people using the whitelist config stuff. The only reason I ask is because it might be confusing to people reading it, and thus beneficial to specify what you mean here.